packages/paste-core/components/timeline/src/TimelineItem.tsx

```diff
@@ -0,0 +1,115 @@
+import { Box } from "@twilio-paste/box";
+import { css, styled } from "@twilio-paste/styling-library";
+import React from "react";
+
+import { TimelineContext, TimelineGroupContext } from "./TimelineContext";
+import { TimelineItemCollapsible } from "./TimelineItemCollapsible";
+import { TimelineItemIcon } from "./TimelineItemIcon";
+import type { TimelineItemProps } from "./types";
+
+const TimelineItem = React.forwardRef<HTMLDivElement, TimelineItemProps>(
+  (
+    { children, icon, timestamp, title, collapsible = false, collapsibleHeading, element = "TIMELINE_ITEM", ...props },
+    ref,
+  ) => {
+    const { orientation } = React.useContext(TimelineContext);
+    const isGrouped = React.useContext(TimelineGroupContext);
+
+    if (!orientation) {
+      throw new Error("TimelineItem must be used within a Timeline component");
+    }
+
+    const ContainerStyled = styled.li(
+      css({
+        "&:last-child>div:first-child::after": {
+          content: "none",
+        },
+        "div::after": {
+          display: isGrouped ? "none" : "block",
```

Comment:
is this needed? I'm inspecting the grouped an non grouped versions for element="TIMELINE_ITEM_ICON_WRAPPER " and in both the ::after has display block?

Reply:
yeah, removing it causes it to look like this:
<img width="279" alt="image" src="https://github.com/user-attachments/assets/080550b1-5349-4705-a4ab-c8930572cebc">